src/content/contributing/release-process/_index.en.md

```diff
@@ -67,7 +71,70 @@ Verify that the build successfully completes as indicated by a green checkmark a
 > https://quay.io/repository/submariner/lighthouse-controller?tab=tags
 > https://quay.io/repository/submariner/lighthouse-coredns?tab=tags
 
-<!-- TODO: Aswin: the openshift-coredns part -->
+## Build Lighthouse CoreDNS
+
+1) Get the coreDNS and change to the folder
+
+```bash
+go get github.com/openshift/coredns
+cd $GOPATH/src/github.com/openshift/coredns
+```
+2) Build the image and push
+
+```bash
+COREDNS_VERSION=<VERSION-NO>
+LH_COREDNS_VERSION=<LH-VERSION-NO>
+COREDNS_IMAGE="lighthouse-coredns:${LH_COREDNS_VERSION}"
+git checkout -b ${COREDNS_VERSION}
+sed -i '/^kubernetes:kubernetes/a lighthouse:github.com/submariner-io/lighthouse/plugin/lighthouse' plugin.cfg
+sed -i '/^github.com/aws/aws-sdk-go/a github.com/submariner-io/lighthouse v0.2.0' go.mod
+sed -i '$a replace\ k8s.io\/apimachinery\ =>\ k8s.io\/apimachinery\ v0.0.0-20190313205120-d7deff9243b1' go.mod
+sed -i '$a replace\ github.com\/openzipkin-contrib\/zipkin-go-opentracing\ =>\ github.com\/openzipkin-contrib\/zipkin-go-opentracing\ v0.3.5' go.mod
+sed -i '$a replace\ github.com/bronze1man/goStrongswanVici\ =>\ github.com/mangelajo/goStrongswanVici\ v0.0.0-20190223031456-9a5ae4453bd' go.mod
```

Comment:
that replacement shouldn't be necessary anymore.

Reply:
Done.